src/sympc/store/crypto_primitive_provider.py

```diff
@@ -110,11 +111,40 @@ def stop_logging(generate_file: bool = False):
             json: returns the json object containing ops details.
         """
         CryptoPrimitiveProvider._LOGGING = False
-        log_json = json.dumps(CryptoPrimitiveProvider._ops_list)
+        log = copy(CryptoPrimitiveProvider._ops_list)
```

Comment:
MAYBE: Use ```CryptoPrimitiveProvider._ops_list.copy()``` (in that case we don't need the import)

Reply:
working

Reply:
done